polyfill/test/duration.mjs

```diff
@@ -935,8 +935,8 @@ describe('Duration', () => {
     const d = new Duration(5, 5, 5, 5, 5, 5, 5, 5, 5, 5);
     const d2 = new Duration(0, 0, 0, 5, 5, 5, 5, 5, 5, 5);
     const relativeTo = Temporal.PlainDateTime.from('2020-01-01T00:00');
-    it('options may only be an object', () => {
-      [null, 1, 'hello', true, Symbol('foo'), 1n].forEach((badOptions) => throws(() => d.round(badOptions), TypeError));
+    it('options may only be an object or string', () => {
```

Comment:
Probably could rename this to be "configuration" here as well and in the other tests below.

Reply:
I'll change this text to the parameter name and let other folks fight about the general name in the next proposal that tries to do required options. 😄